microsoft-edge/webview2/releasenotes.md

```diff
@@ -47,6 +47,7 @@ Release Date: November 23, 2020
 
 * Enabled WinForms designer in .NET Core 3.1+ and .NET 5.  
 * Improved .NET cookie management.  \([\#611][GithubMicrosoftedgeWebviewfeedbackIssue611]\).  
+* Renamed CoreWebView2Ready to [CoreWebView2InitializationCompleted](https://docs.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2initializationcompletedeventargs?view=WebView2-dotnet-1.0.707-prerelease).
```

Comment:
@pagoe-msft  can you fix the link please? Thanks.

Reply:
link should be relative path?

Reply:
Updated link and verified that the link exists on the live branch.